server/cron/calendar_cron_test.go

```diff
@@ -289,12 +289,12 @@ func TestCalendarEventsMultipleHosts(t *testing.T) {
 			},
 			{
 				HostID:  hostID3,
-				Email:   userEmail3,
+				Email:   "", // because it does not belong to example.com
```

Comment:
Nit. Blank email is a bit confusing. Would be nicer to return a boolean flag if email is valid.

Reply:
Ah, that would require modifying the implementation, which would require re-test :)
Maybe for the next iteration?